pkg/cidata/cidata.TEMPLATE.d/boot/30-install-packages.sh

```diff
@@ -71,6 +71,34 @@ elif command -v dnf >/dev/null 2>&1; then
 		# Workaround for https://github.com/containerd/stargz-snapshotter/issues/340
 		ln -s fusermount3 /usr/bin/fusermount
 	fi
+elif command -v yum >/dev/null 2>&1; then
+    echo "DEPRECATED: CentOS7 and others RHEL-like version 7 are unsupported and might be removed in future lima releases"
```

Comment:
Indent seems broken.
Also, could you squash commits?

Reply:
Will do.